File: llm_outputs/codellama7b/GuruSamples/csharp/replace-method-with-method-object_refactored.cs
public class Order 
{
  //...
  public double Price() 
  {
    double primaryBasePrice = ComputeBasePrice(primary);
    double secondaryBasePrice = ComputeBasePrice(secondary);
    double tertiaryBasePrice = ComputeBasePrice(tertiary);
    // Perform long computation.
  }

  private double ComputeBasePrice(OrderItem item)
  {
    // Perform long computation.
  }
}

File: llm_outputs/codellama13b/GuruSamples/java/replace-method-with-method-object_refactored.java
class Order {
  //...
  public double price() {
    double primaryBasePrice = computeBasePrice();
    double secondaryBasePrice = computeBasePrice();
    double tertiaryBasePrice = computeBasePrice();
    // Perform long computation.
  }

  private double computeBasePrice() {
    // Perform long computation.
    return basePrice;
  }
}